test/e2e/resources/components/roRegex.brs

```diff
@@ -0,0 +1,21 @@
+sub main()
+    regex = createObject("roRegex", "hello_[0-9]*_world", "i")
+
+    print "HeLlO_123_WoRlD is match of hello_[0-9]*_world: " regex.ismatch("HeLlO_123_WoRlD")
+    print "goodbye_123_WoRlD isn't match of hello_[0-9]*_world: " regex.ismatch("HeLlO_123_WoRlD")
+
+    regex = createObject("roRegex", ",", "i")
+
+    print "Replacing ',' in 2019,03,26 by '-' on first occurrence: " regex.replace("2019,03,26", "-")
+    print "Replacing ',' in 2019,03,26 by '-' on all occurrences: " regex.replaceall("2019,03,26", "-")
+    print "Split by ','" regex.split("2019,03,26")
+
+    regex = createObject("roRegex", "\d+", "i")
+
+    print "First match" regex.match("123 456 789")
+    matches = regex.matchall("123 456 789")
+    print "All matches: "
+    print matches[0]
+    print matches[1]
+    print matches[2]
+end sub
```

Comment:
Great test file!  I don't see its output getting compared to anything though so it might not be getting executed 😅 We'll need something like this:

https://github.com/sjbarag/brs/blob/e2168603cda83b746a7101f2bb7f782b0178ce5f/test/e2e/BrsComponents.test.js#L53-L62

Reply:
My bad, didn't see this. The E2E is being executed now 😄